python/dgl/graphbolt/link_data_format.py

```diff
@@ -0,0 +1,23 @@
+"""Linked data format."""
```

Comment:
Rename the file to data_format.py
```suggestion
"""Data format enums for graphbolt."""
```